kong/plugins/acl/schema.lua

```diff
@@ -4,7 +4,8 @@ return {
   no_consumer = true,
   fields = {
     whitelist = { type = "array" },
-    blacklist = { type = "array" }
+    blacklist = { type = "array" },
+    hide_groups_header = { type = "boolean", default = false },
```

Comment:
this needs another PR against `next`, to convert existing records from `nil` to `false`. Since the schema says the default is `false`, it is expected to always be a boolean value, and not an absent one. To keep the data in line with the schema.

Reply:
Is it problematic for this to go into the 0.14.1 minor version as is? I think in my testing of custom plugins the addition of new schema fields are treated as nil which will at least produce the desired results until converting nil to false right? For some reason initially I was thinking Kong core recognized those defaults and would take action on populating them by default in all plugins it sees, maybe a nice future Kong enhancement to automate that process.

![image](https://user-images.githubusercontent.com/31913027/44155648-deee69f4-a07b-11e8-8aa2-6dea81545d35.png)